### PR TITLE
Update README with docker information

### PR DESCRIPTION
**Pull request by hmunfru**
#### REVIEWERS
@flopezag 

#### DESCRIPTION
- Update documentation with docker information.
- Remove the install testbed dockers

#### TESTING
Done

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -430,6 +430,28 @@ code. Keep in mind that it requires python2.7 to execute the unit tests.
      pip install -r test-requirements.txt
      nosetests --with-coverage --cover-package=./ --exe
 
+Unit tests with Docker execution
+********************************
+Skuld unit tests can be executed by docker. To do that, firtly it is required to create the docker image, with the
```
Comment (flopezag) [inline README.rst]: firstly, it is required the creation of ...

File: README.rst [hunk 2]
```diff
@@ -480,6 +502,35 @@ advantage is that the script support OS_TRUST_ID, while other tools as nova does
 
 Top_
 
+Acceptance tests with Docker execution
+**************************************
+Skuld acceptance tests can be executed by Docker. To do that, firtly it is required to create the docker image.
```
Comment (flopezag) [inline README.rst]: idem

File: README.rst [hunk 3]
```diff
@@ -480,6 +502,35 @@ advantage is that the script support OS_TRUST_ID, while other tools as nova does
...
+To do that:
+
+.. code::
+
+    docker build -t fiware-skuld -f docker/Dockerfile docker
+
+Once the fiware-skuld image is created, we can run it by using docker-compose (to include the environment variables). To export then
```
Comment (flopezag) [inline README.rst]: I do not understand the sentence:

To export then is required:

File: README.rst [hunk 4]
```diff
@@ -480,6 +502,35 @@ advantage is that the script support OS_TRUST_ID, while other tools as nova does
...
+    export OS_PASSWORD = {the password}
+    export OS_REGION_NAME = {the region}
+    export OS_PROJECT_DOMAIN_NAME = {the project domain name}
+    export OS_USER_DOMAIN_NAME = {the user domain name}
+    docker-compose -f docker/docker-compose.yml up
+
+When docker has finished, you can obtain the tests results by
+.. code::
+
+   docker cp docker_fiwareskuld_1:/opt/fiware-skuld/tests/acceptance/testreport .
```
Comment (flopezag) [inline README.rst]: docker_fiwareskuld_1 or docker_fiware?

File: README.rst [hunk 5]
```diff
@@ -480,6 +502,35 @@ advantage is that the script support OS_TRUST_ID, while other tools as nova does
...
+
+Top_
```
Comment (flopezag) [inline README.rst]: Two things:
- don't you need to have the OpenStack testbed installed in order to test the Skuld component?
- Have you check both unit tests and acceptance tests?

**Comment by flopezag** [comment]
Put reference to the PR in the fiware-test-deploy in which you put these docker files...

File: README.rst (hunk 4 above)
Comment (hmunfru) [inline README.rst]: docker_fiwareskuld_1 verfied. It correspond to the contanair name

File: README.rst (hunk 5 above)
Comment (hmunfru) [inline README.rst]: NO really. YOu need a testbed, you can use FIWARE lab, your own Openstack testbed, or a testbed deployed by the fiware-testbed-deploy scripts. I can explain a bit more that you need a Openstack testbed to be able to test.

Comment (hmunfru) [inline README.rst]: What do you mean by have you check it?? These docker are executed everyday by jenkins

**Comment by hmunfru** [comment]
Docker files are in the docker folder, not in fiware-testbed-deploy. The reference to the fiware-deploy-testbed is already there.

File: README.rst [hunk 6]
```diff
@@ -480,6 +502,36 @@ advantage is that the script support OS_TRUST_ID, while other tools as nova does
...
+.. code::
+
+    docker build -t fiware-skuld -f docker/Dockerfile docker
+
+Once the fiware-skuld image is created, we can run it by using docker-compose (exporting the right
+environment variables).
+
+.. code::
+
+    export OS_AUTH_URL = {the auth uri of the testbed agains the tests are going to be execute}
```
Comment (flopezag) [inline README.rst]: What is the testbed and where is it?

File: README.rst (hunk 4 above)
Comment (flopezag) [inline README.rst]: NTC

File: README.rst (hunk 5 above)
Comment (flopezag) [inline README.rst]: Yes, I think that you should explain more it, what is a testbed and where is it or how to create it.

**Comment by flopezag** [comment]
No, no, you do not understand me, this PR is deleting docker file related to the testbed, so I mentioned that you add the link to the corresponding PR in the oder repository in which you add those files.

**Comment by flopezag** [comment]
LGTM

**Pull request merged**